packages/patrol/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.8.1
+
+- Fix MacOSAutomator
```

Comment:
Please let's link some issue and give more info here than just `Fix MacOSAutomator` so that when we look at it in a year, we know what happened :)

Reply:
There's no issue for that. Should I create new one?

Reply:
I think we can link this PR here. Also we used to write (Unreleased) next to version if it's not released yet